contrib/opencensus-ext-azure/opencensus/ext/azure/common/__init__.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, *args, **kwargs):
         logging_sampling_rate=1.0,
         max_batch_size=100,
         minimum_retry_interval=60,  # minimum retry interval in seconds
-        proxy=None,
+        proxies='{}',  # string maps url schemes to the url of the proxies
```

Comment:
I don't think I was clear in my previous explanation. You keep proxies=None in the constructor, but in `process_options` you check if proxies is None and you set it to `"{}"` in that function.